src/client/opamCommands.ml

```diff
@@ -408,17 +416,31 @@ let init cli =
         OpamStd.Sys.guess_dot_profile shell >>| OpamFilename.of_string)
     in
     let cygwin_setup =
-      match cygwin_internal, cygwin_location with
-      | `internal, Some _ ->
+      match cygwin_internal, cygwin_location, cygwin_extra_packages with
+      | `internal , Some _, _ ->
         OpamConsole.error_and_exit `Bad_arguments
           "Options --cygwin-internal-install and \
            --cygwin-location are incompatible";
-      | `no, Some _ ->
-        OpamConsole.note "Ignoring argument --cygwin-location";
-        Some `no
-      | `none, None -> None
-      | (`default_location | `none), Some dir -> Some (`location dir)
-      | (`internal | `default_location | `no) as setup, None -> Some setup
+      | `default_location, _, Some _ ->
+        OpamConsole.error_and_exit `Bad_arguments
+          "Options --cygwin-location-install and \
```

Comment:
```suggestion
          "Options --cygwin-location and \
```

Reply:
Indeed but it is `local` we need to change to have `--cygwin-local-install`.